api-client/src/models/PoiModel.ts

```diff
@@ -81,6 +88,30 @@ class PoiModel extends ExtendedPageModel {
     }
   }
 
+  get openingHours(): OpenHoursModel[] | null {
+    return this._openingHours
+  }
+
+  get temporarilyClosed(): boolean {
+    return this._temporarilyClosed
+  }
+
+  get isCurrentlyOpened(): boolean {
+    if (!this.openingHours) {
+      return false
+    }
+    // isoWeekday return 1-7 for the weekdays
+    const weekday = moment().isoWeekday() - 1
+    const dateFormat = 'LT'
+    const currentTime = moment().locale('de').format(dateFormat)
+
+    return (
+      this.openingHours[weekday]?._timeSlots.some(timeslot =>
+        moment(currentTime, dateFormat).isBetween(moment(timeslot.start, dateFormat), moment(timeslot.end, dateFormat))
+      ) || (this.openingHours[weekday]?.allDay as boolean)
+    )
```

Comment:
I don't agree with that, because we just get one singleValue for the `openingHours[]` if we put it into the `OpeningHoursModel` we have to check every entry of this array or pass to the `OpeningHoursModel` the entire `openingHours[]` which doesn't make sense

Reply:
Oh true. Yeah nvm, sorry :facepalm: